lib/graphql/analysis/analyze_query.rb

```diff
@@ -21,7 +26,11 @@ def analyze_query(query, analyzers)
         reduce_node(op_node, analyzers_and_values)
       end
 
-      analyzers_and_values.map { |(r, value)| finalize_reducer(r, value) }
+      if !@errors.blank?
```

Comment:
`.blank?` and `.present?` are ActiveSupport methods, right? We want to support contexts _without_ ActiveSupport, so I think we should switch these  to `.none?` and `.any?`, respectively.